src/utilities/hash.ts

```diff
@@ -0,0 +1,59 @@
+//https://stackoverflow.com/questions/40958727/javascript-generate-unique-number-based-on-string
```

Comment:
If this file is intended to only be for external events, then can we rename the file to reflect that? If you would like to continue using `cyrb53a()`, then it can stay in this file as it seems generic enough, but then please extract the external event specific logic out to its own util file.

Reply:
This file has now been removed!